docs/user-guide/recipes.rst

```diff
@@ -38,7 +38,8 @@ v2 registries.
    
     ``registry-1.docker.io`` is a Docker V2 Registry API. For V1 API
     ``index.docker.io`` should be used, along with ``--enable-v1 true`` and
-    ``--enable-v2 false``.
+    ``--enable-v2 false``, however V1 content is deprecated and Docker
```

Comment:
It might read a little better if you end the last sentence and start a new one. Maybe s/, however/./?